website/sidebars.json

```diff
@@ -145,7 +145,8 @@
       ]
     },
     "implementation_guides/workshops",
-    "faq"
+    "faq",
+    "telemetry"
```

Comment:
Last one, I think this is best put as the last item in the "docs" section.

Reply:
you mean on the "Pact Docs" sidebar don't you?